openqa-investigate

```diff
@@ -193,4 +200,4 @@ main() {
     exit "$error_count"
 }
 
-main "$@"
+[[ ${TESTING:-} ]] || main "$@"
```

Comment:
To avoid the extra variable, this _could_ be replaced by
```
caller 0 >/dev/null || main "$@"
```
but I admit that the documentation is not very clear on that.

Reply:
I'm also a fan of putting functions into a "library" (here `_common`) and source this instead, but admittedly in bash that can be more complicated because of global variables

Reply:
I'd like the caller approach. For the second suggestion Martchus and me decided to keep the functions in this file as long as the functions are actually not shared with other scripts